src/main/java/com/yahoo/bullet/record/BulletRecord.java

```diff
@@ -259,6 +259,72 @@ public BulletRecord setDoubleMap(String field, Map<String, Double> object) {
         return set(field, object);
     }
 
+    /**
+     * Insert a List of Boolean field.
+     *
+     * @param field The non-null name of the field.
+     * @param object The value to insert.
+     * @return This object for chaining.
+     */
+    public BulletRecord setBooleanList(String field, List<Boolean> object) {
```

Comment:
Rename these new methods to start with the outermost type to keep the convention? ```setListOfBoolean```? 

I guess we have setMapOfIntegerMap instead of setMapOfMapOfInteger so we already kind of violate it...

Reply:
I'm not sure since we have setBooleanMap, setIntegerMap, etc.

Reply:
Ok let's leave it as you have it